js/common.js

```diff
@@ -0,0 +1,7 @@
+const handleCountryClick = (country, event) => {
+    event.preventDefault();
+    
+    localStorage.setItem('selectedCountry', JSON.stringify(country));
+    window.location.href = './details.html';
+    
```

Comment:
1 - please extract your local storage logic into a service, this way you wont repeat yourself and future changes will only take place in one file

2 - Do not use hardcoded strings, save those into constants